frontend/src/components/StocksList.tsx

```diff
@@ -19,8 +25,9 @@ const StocksList: React.FC = () => {
     }, []);
 
     return (
+<AuthenticatedTemplate>
         <div>
-            <h2 className="text-lg font-bold mb-2 mt-2">Liste des stocks</h2>
+            <h2 className="text-lg font-bold mb-2 mt-2">Liste des stocks {msalInstance.accounts[0].username}</h2>
```

Comment:
suggestion: use a wrapper around the msal instance to hide the .account[0].username.

Reply:
à voir sur la branche front